README.md

```diff
@@ -173,13 +173,19 @@ go test -v ./...
-| `KNUU_BUILDER`              | The builder to use for building images.                                                                                                                                                                                                      | `docker`, `kubernetes`           | `docker` |
-| `KNUU_DEDICATED_NAMESPACE`  | Creates a namespace for the test, and it gets deleted after `KNUU_TIMEOUT`.<br/> *This ENV var has priority over `KNUU_NAMESPACE`*.<br/>The namespace uses a prefix (`knuu-`) + the unique id as name, example: `knuu-20240404-102053-681`   | `true`, `false`                  | `false`  |
-| `DEBUG_LEVEL`               | The debug level.                                                                                                                                                                                                                             | `debug`, `info`, `warn`, `error` | `info`   |
+| Environment Variable        | Description                                                                                                     | Possible Values                  | Default  |
+|-----------------------------|-----------------------------------------------------------------------------------------------------------------|----------------------------------|----------|
+| `KNUU_TIMEOUT`              | The timeout for the tests.                                                                                      | Any valid duration               | `60m`    |
+| `KNUU_NAMESPACE`            | The namespace where the instances will be created.                                                              | Any valid namespace name         | `test`   |
+| `KNUU_BUILDER`              | The builder to use for building images.                                                                         | `docker`, `kubernetes`           | `docker` |
+| `KNUU_DEDICATED_NAMESPACE`  | Creates and deletes a dedicated namespace for the test after `KNUU_TIMEOUT`. See note below for more details.*	 | `true`, `false`                  | `false`  |º
+| `DEBUG_LEVEL`               | The debug level.                                                                                                | `debug`, `info`, `warn`, `error` | `info`   |
+
+* Note on KNUU_DEDICATED_NAMESPACE: 
+
+When set to true, this environment variable tells knuu to create a unique namespace for running tests, which will be automatically deleted after the specified timeout period (`KNUU_TIMEOUT`).
+The created namespace will have a prefix of `knuu-` followed by a unique identifier (unique id of the test), which ensures that each test run has its isolated environment within the Kubernetes cluster.
+The unique identifier is a timestamp autogenerated string that uniquely identifies each test run, for example, `knuu-20240404-102053-681`.
```

Comment:
Sorry for beeing picky, it's not always random:
```
func InitializeWithIdentifier(uniqueIdentifier string) error
```

https://pkg.go.dev/github.com/celestiaorg/knuu@v0.12.0/pkg/knuu#InitializeWithIdentifier